operator/cmd/mesh/testdata/operator/output/operator-init.yaml

```diff
@@ -162,11 +162,11 @@ spec:
         status:
           description: 'Status describes each of istio control plane component status at the current time.
             0 means NONE, 1 means UPDATING, 2 means HEALTHY, 3 means ERROR, 4 means RECONCILING.
-            More info: https://github.com/istio/operator/blob/master/pkg/apis/istio/v1alpha2/v1alpha2.pb.html &
+            More info: https://github.com/istio/api/blob/master/operator/v1alpha1/istio.operator.v1alpha1.pb.html &
```

Comment:
This should actually now point to istio/api/operator

Reply:
Yes, it is https://github.com/istio/api/blob/master/operator/v1alpha1/istio.operator.v1alpha1.pb.html